src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -71,12 +58,10 @@ private int executeQuery(String query) throws InterruptedException, IOException
   }
 
   public boolean verifyCmekKey(String tableName, String cmekKey) throws IOException {
```

Comment:
Can we also make this utility method static?

Reply:
done